src/components/fieldset/template.njk

```diff
@@ -1,15 +1,15 @@
 {% from "error-message/macro.njk" import govukErrorMessage -%}
 
-<fieldset class="govuk-c-fieldset
+<fieldset class="govuk-c-fieldset {%- if params.error.active %} govuk-c-fieldset--error{% endif %}
   {%- if params.classes %} {{ params.classes }}{% endif %}"{% for attribute, value in params.attributes %} {{ attribute }}="{{ value }}"{% endfor %}>
   {% if params.legendHtml or params.legendText %}
   <legend class="govuk-c-fieldset__legend">
     {{ params.legendHtml | safe if params.legendHtml else params.legendText }}
     {% if params.legendHintText or params.legendHintHtml %}
     <span class="govuk-c-fieldset__hint">{{ params.legendHintHtml | safe if params.legendHintHtml else params.legendHintText }}</span>
     {% endif %}
-    {% if params.errorMessage %}
-    {{ govukErrorMessage(params.errorMessage) | trim | indent(4) -}}
+    {% if params.error.active %}
```

Comment:
This will prevent passing an `errorMessage` directly, without specifying an additional `error.active`. I think we should be flexible in this matter and allow an `errorMessage` without additional params.